### ISSUE TITLE
Can't  create more than 125 container 

### ISSUE THREAD

**Issue by Dinaamagdy** [g8os/core0#242]
## Steps 
- create more than 125 container 
## Issue 
![selection_342](https://cloud.githubusercontent.com/assets/24386802/26066965/ed552f3c-3998-11e7-9575-ea58f3034823.png)

## SV
56c40ef13cb166ca15560bc3280c3413c06d51a7

**Comment by muhamadazmy** [comment]
There was an issue of how we monitor the filesystem for un-mount which uses inotify and the system had a limit on number of watchers we can create. We changed the implementation to use a different method. but then we were hit by another limitations (number of open files per process).

We changed the upper limit of core0, and set a hard limit on the number of containers u can have running on the system to 1000.
